stdlib/universe/group.go

```diff
@@ -150,7 +182,7 @@ type groupTransformation struct {
 	keys []string
 }
 
-func NewGroupTransformation(spec *GroupProcedureSpec, id execute.DatasetID, mem *memory.Allocator) (execute.Transformation, execute.Dataset) {
+func NewGroupTransformation(ctx context.Context, spec *GroupProcedureSpec, id execute.DatasetID, mem *memory.Allocator) (execute.Transformation, execute.Dataset) {
```

Comment:
Can you change this to return an error too so that the error from `NewGroupTransformation` can be standardized?